.github/workflows/dependabot-pnpm-lock.yml

```diff
@@ -0,0 +1,46 @@
+name: Update pnpm-lock.yaml for Dependabot PRs
+
+on:
+  pull_request_target:
+    types: [opened, synchronize]
+    paths:
+      - 'package.json'
```

Comment:
redundant, as you already got **/package.json

Reply:
But would **/package.json match the root level one?

Reply:
Ah, you are right: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet